tests/unit/s2n_ecc_evp_test.c

```diff
@@ -24,21 +24,31 @@
 
 #define ECDHE_PARAMS_LEGACY_FORM 4
 
+const static struct s2n_ecc_named_curve *const supported_curves_list[] = {
```

Comment:
It's not great to duplicate lists like this, even for testing. Can you use the same list for both the code and the test?

Reply:
We could, I was trying to keep the hard-coded list from being accessed in the logic flow outside of testing. I will get rid of the duplicate.